kotlin/browse-building-floors/src/main/java/com/esri/arcgisruntime/sample/browsebuildingfloors/MainActivity.kt

```diff
@@ -66,62 +65,58 @@ class MainActivity : AppCompatActivity() {
 
         map.addDoneLoadingListener {
             if (map.loadStatus == LoadStatus.LOADED && map.floorDefinition != null) {
-
                 // get and load the floor manager
-                val floorManager = map.floorManager
-                floorManager.loadAsync()
-
-                // set initial floor level to currentFloor
+                val floorManager = map.floorManager.apply { loadAsync() }
+                // set initial floor level to ground floor
```

Comment:
I think this was better before. Remember we're trying to use the `apply` block as an extended builder.

someObject(some paramater).apply {
    someOtherParamater = blah
    yetSome Other = blah 2
}

Let's reserve things like .loadAsync() and .cancel() to calls to the parent object, just for readability